skywalking-collector/skywalking-collector-cluster/src/main/java/com/a/eye/skywalking/collector/actor/AbstractMember.java

```diff
@@ -0,0 +1,57 @@
+package com.a.eye.skywalking.collector.actor;
+
+import akka.actor.ActorRef;
+import com.a.eye.skywalking.collector.actor.selector.WorkerSelector;
+import com.a.eye.skywalking.collector.cluster.WorkersRefCenter;
+import com.a.eye.skywalking.collector.queue.MessageHolder;
+import com.lmax.disruptor.EventHandler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+/**
+ * @author pengys5
+ */
+public abstract class AbstractMember<T> implements EventHandler<MessageHolder<T>> {
+
+    private Logger logger = LogManager.getFormatterLogger(AbstractMember.class);
+
+    private ActorRef actorRef;
+
+    public ActorRef getSelf() {
+        return actorRef;
+    }
+
+    public AbstractMember(ActorRef actorRef) {
+        this.actorRef = actorRef;
+    }
+
+    /**
+     * Receive the message to analyse.
+     *
+     * @param message is the data send from the forward worker
+     * @throws Exception is the exception thrown by that worker implementation processing
+     */
+    public abstract void receive(Object message) throws Exception;
```

Comment:
No need to use `Object.class`. This forces the implementations using force type-cast. Recommendation:

```java
public abstract void receive(T message) throws Exception
```